custom_components/my_fitnesspal/__init__.py

```diff
@@ -21,7 +21,7 @@
 from .const import DOMAIN, STARTUP_MESSAGE, PLATFORMS, ATTRIBUTION
 
 
-SCAN_INTERVAL = timedelta(minutes=30)
+SCAN_INTERVAL = timedelta(minutes=5)
```

Comment:
The only thing about the scan interval is that I was afraid MyFitnessPal would somehow notice the pulling. But we can try 5 minutes.

Reply:
I haven't had any issues will pulling once every 5 mins for the past 2 weeks, but feel free to bump this if needed 👍